pages/report-hazard/style.css

```diff
@@ -75,15 +73,128 @@ img {
   border-radius: 0 0 1rem 1rem;
 }
 
+/* Header */
+#report-hazard-body main {
+  height: 100%;
+}
+
+header {
+  display: none;
+}
+
+/* Back button */
+#backButton {
+  border: none;
+  background: none;
+  color: inherit;
+  cursor: pointer;
+  text-decoration: none;
+  color: var(--neutral-neutral-500, #667185);
```

Comment:
this should be

color: var(--neutral-500);

Reply:
Done